test.js

```diff
@@ -366,6 +366,31 @@ test('should serialize custom error with `.toJSON` defined with `serializeError`
 	t.not(stack, undefined);
 });
 
+test('should ignore `.toJSON` methods if set in the options', t => {
+	class CustomError extends Error {
+		constructor() {
+			super('foo');
+			this.name = this.constructor.name;
+			this.value = 10;
+		}
+
+		toJSON() {
+			return {
+				message: this.message,
+				amount: `$${this.value}`,
+			};
+		}
+	}
```

Comment:
```suggestion
	}

```

Reply:
Too cramped :P

Reply:
I just copy-pasted your code 😎 

![C-658VsXoAo3ovC](https://user-images.githubusercontent.com/1402241/161581607-89c928bb-db11-4342-9f84-6ade5a2b9d1d.jpg)

Reply:
Not mine: https://github.com/sindresorhus/serialize-error/commit/34655eb0d57178ba27cb8e7992df3f7182b74e1d#diff-13876b4beb64b9f156474dc78f9c923952a7ca210d4507b6b3135bbe244f8a60R219-R239

Reply:
Who reviewed and merged https://github.com/sindresorhus/serialize-error/pull/38? 😜 

![image](https://user-images.githubusercontent.com/1402241/161647042-7eef4d25-9350-43b5-90a4-6da2242aa9cc.jpeg)